SmartVision/WindowsForms/FormControl/FormFaceDetection.cs

```diff
@@ -233,6 +231,54 @@ private void disableScanPanel()
             }
         }
 
+        private Api.Models.MissingPerson InitializeMissingPerson()
+        {
+            Api.Models.MissingPerson missingPerson = new Api.Models.MissingPerson();
+            missingPerson.faceToken = "0.777";
```

Comment:
Leave it blank

Reply:
Yeah, I chose this as a temporary solution until we store face tokens. No worries, now removed.